### PR TITLE
no more block mined points

### PR DESCRIPTION
**Pull request by lwisne**
## Summary
Creates a config var for counting block mined points. Set it to true for test to minimize the overall set of changes to tests etc. Otherwise default to false. 

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: .env.test [hunk 1]
```diff
@@ -1,3 +1,4 @@
+ALLOW_BLOCK_MINED_POINTS=true
```
Comment (rohanjadvani) [inline .env.test:1]: Can this be added to [`.env.template`](https://github.com/iron-fish/ironfish-api/blob/master/.env.template) too?

File: src/events/events.service.spec.ts [hunk 2]
```diff
@@ -896,6 +896,15 @@ describe('EventsService', () => {
         expect(event).toBeNull();
       });
     });
+
+    describe('respects enviornment variables', () => {
```
Comment (rohanjadvani) [inline src/events/events.service.spec.ts:900]:
```suggestion
    describe('when `ALLOW_BLOCK_MINED_POINTS` is false', () => {
```

File: src/events/events.service.spec.ts [hunk 3]
```diff
@@ -896,6 +896,15 @@ describe('EventsService', () => {
...
+      it('returns null if config returns false for get ALLOW_BLOCK_MINED_POINTS', async () => {
```
Comment (rohanjadvani) [inline src/events/events.service.spec.ts:901]:
```suggestion
      it('returns null', async () => {
```

**Review by rohanjadvani — APPROVED** [review]

File: .env.test (hunk 1 above)
Comment (lwisne) [reply .env.test:1]: yup, done

**Pull request merged**